display_list/dl_builder.h

```diff
@@ -52,7 +56,7 @@ class DisplayListBuilder final : public virtual DlCanvas,
   void Save() override;
 
   // |DlCanvas|
-  void SaveLayer(std::optional<const DlRect>& bounds,
+  void SaveLayer(const std::optional<DlRect>& bounds,
```

Comment:
Why was this changed?

Reply:
const in an optional doesn't really prevent you from changing it, the optional itself has to be const. And at that point, why have the const inside the template type?

Reply:
> why have the const inside the template type?

To make sure no one modifies the `DlRect`.  I would have expected `const std::optional<const DlRect>& bounds`